plugins/azure_ad_admin/icon_azure_ad_admin/actions/update_user_info/schema.py

```diff
@@ -48,7 +48,7 @@ class UpdateUserInfoInput(komand.Input):
     "job_title": {
       "type": "string",
       "title": "Job Title",
-      "description": "The user’s job title",
+      "description": "The user is job title",
```

Comment:
Shouldn't that be same as before?
```suggestion
      "description": "The user's job title",
```